shell/platform/android/io/flutter/embedding/engine/renderer/FlutterRenderer.java

```diff
@@ -195,6 +209,11 @@ protected void finalize() throws Throwable {
         super.finalize();
       }
     }
+
+    @Override
+    public void setImageFrameListener(@Nullable ImageFrameListener listener) {
```

Comment:
I don't think this is used or related to bug. If this is indeed the case, please remove.